### PR TITLE
revamped tool page

### PR DESCRIPTION
**Pull request by owocki**
# What

* adds a clickable table of contents
* adds faucet
* adds gitcoinbot
* adds coming soon tools

screencapture-localhost-8080-toolbox-2018-03-20-20_30_46

### CODE REVIEW

**Comment by Elaniobro** [comment]
@owocki overall the visual style to me looks great. I do have a personal issue with `<ol> || <li>` lists in general. I dislike how the `#` maintains the same font-weight/size as the actual text! 

Would you consider making the list counter smaller? Here is a neat css trick I learned recently to do so, that gives you more finite control over the `<ol> || <ul>` markers. I would also be interested in @PixelantDesign design input 😸 
```
ol {
  list-style: none;
  counter-reset: listNumbering;
}

li {
  counter-increment: listNumbering;
}

li::before {
  content: counter(listNumbering);
  font-size: XXpx/rem/em/pt
}
```

or sass/less
```
ol {
  list-style: none;
  counter-reset: listNumbering

  li {
    counter-increment: listNumbering;

    &::before{
      content: counter(listNumbering);
      font-size: XXpx/rem/em/pt
    }
  }
}
```

File: app/assets/v2/css/toolbox.css [hunk 1]
```diff
@@ -25,7 +25,46 @@
...
+}
+
+
+#toc{
+  position: fixed;
+  top: 150px;
+  left: 50px;
+  border-top: 1px black solid;
+  border-left: 2px black solid;
+  background-color: #f5f5f5;
```
Comment (Elaniobro) [inline app/assets/v2/css/toolbox.css]: Personal preference is to use all uppercase: `#F5F5F5` I find it helps with legibility

File: app/assets/v2/css/toolbox.css [hunk 2]
```diff
@@ -25,7 +25,46 @@
...
+
+@media (max-width: 1100px) {
+  #toc{
+    left: 10px;
+  }
+}
+
 @media (max-width: 991px) {
+  #toc{
+    background-color: white;
```
Comment (Elaniobro) [inline app/assets/v2/css/toolbox.css]: Stay consistent: `#FFFFFF`

File: app/assets/v2/css/toolbox.css [hunk 3]
```diff
@@ -25,7 +25,46 @@
...
+    position: initial;
+    border-top: 0px;
+    border-left: 0px;
+    padding: 10px 5px;
+  }
+  #toc a.active{
+    color:#8e8e8e;
+  }
   .body {
     padding-left: 60px !important;
```
Comment (Elaniobro) [inline app/assets/v2/css/toolbox.css]: what? why? how? no! `!important`

File: app/assets/v2/js/toolbox.js [hunk 4]
```diff
@@ -24,4 +33,21 @@ $(document).ready(function() {
...
+    var scrollPos = $(document).scrollTop();
+
+    if (parseInt(scrollPos) % 100 < 10) {
+      $('#toc a').removeClass('active');
+      $('#toc a').each(function() {
+        var href = $(this).attr('href');
+        var target_selector = href;
+
+        if ($(target_selector).isInViewport()) {
+          if ($('toc a.active').length < 1) {
```
Comment (Elaniobro) [inline app/assets/v2/js/toolbox.js]: Missing the `#` selector

File: app/dashboard/templates/toolbox.html [hunk 5]
```diff
@@ -43,8 +43,19 @@ <h2 class="title">Toolbox</h2>
                 </div>
               </div>
             </div>
+
+            <div id="toc">
```
Comment (Elaniobro) [inline app/dashboard/templates/toolbox.html]: We should consider removing `id` all together and using strictly class names. It allows for better .css inheritance if gitcoin decides to use the BEM method. Just my $0.02wei

File: app/dashboard/views.py [hunk 6]
```diff
@@ -775,74 +721,104 @@ def toolbox(request):
...
-          },
-          {
-              "name": "iOS app",
-              "img": static("v2/images/tools/iOS.png"),
-              "description": '''Gitcoin has an iOS app in alpha. Install it to
-                browse funded work on-the-go.''',
-              "link": reverse("ios"),
+          },{
+              "name": "gitcoinbot",
+              "img": static("v2/images/helmet.png"),
```
Comment (Elaniobro) [inline app/dashboard/views.py]: is this missing the relative `/` in the beginning or is it add else where? If the former should it then be: `static("/v2/images/helmet.png")`

Comment (mbeacom) [reply app/dashboard/views.py]: The slash isn't required here.

File: app/assets/v2/css/toolbox.css [hunk 7]
```diff
@@ -25,7 +25,46 @@
   font-size: 1.4em;
 }
 
+#Faucet img {
+  margin-top: 40px;
+}
+
+
```
Comment (thelostone-mc) [inline app/assets/v2/css/toolbox.css]: Can we remove the extra lines?

File: app/assets/v2/css/toolbox.css [hunk 8]
```diff
@@ -25,7 +25,46 @@
...
+#toc{
+  position: fixed;
+  top: 150px;
+  left: 50px;
+  border-top: 1px black solid;
+  border-left: 2px black solid;
+  background-color: #f5f5f5;
+  padding: 10px 5px;
+}
+#toc li,
```
Comment (thelostone-mc) [inline app/assets/v2/css/toolbox.css]: Add an empty line?

**Comment by owocki** [comment]
> Would you consider making the list counter smaller? 

tried this and it does look better.. thanks!

File: app/assets/v2/js/toolbox.js (hunk 4 above)
Comment (owocki) [reply app/assets/v2/js/toolbox.js]: thx

File: app/assets/v2/css/toolbox.css (hunk 1 above)
Comment (owocki) [reply app/assets/v2/css/toolbox.css]: word we should perhaps add this to the style guide

File: app/assets/v2/css/toolbox.css (hunk 7 above)
Comment (owocki) [reply app/assets/v2/css/toolbox.css]: removed!

File: app/assets/v2/css/toolbox.css (hunk 8 above)
Comment (owocki) [reply app/assets/v2/css/toolbox.css]: not sure i follow ..?

**Pull request merged**